doc/oper.xml

```diff
@@ -1503,3 +1503,41 @@ true
   </Description>
 </ManSection>
 <#/GAPDoc>
+  
+<#GAPDoc Label="HamiltonianPath">
+<ManSection>
+  <Prop Name="HamiltonianPath" Arg="digraph"/>
+  <Returns>A List or <K>fail</K>.</Returns>
+  <Description>
+    Returns a <E>Hamiltonian</E> path if one exists, <K>fail</K> if not.
```

Comment:
Only emphasis words when you are defining them, here you are not defining this, so lose the emphasis.